File: postes.py
#!/usr/bin/env python3.7

# Guilherme A. Pinto, OBI-2017, postes

[N] = [int(c) for c in input().split()]

substituidos = 0
consertados = 0

X = [int(c) for c in input().split()]

for i in range(N):
    if ( X[i] < 50 ):
        substituidos += 1
    if ( 50 <= X[i] < 85 ):
        consertados += 1

print(substituidos, consertados)

File: postes.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;
    
    int substituidos = 0;
    int consertados = 0;
    
    for (int i = 0; i < N; i++) {
        int altura;
        cin >> altura;
        
        if (altura < 50) {
            substituidos++;
        }
        if (altura >= 50 && altura < 85) {
            consertados++;
        }
    }
    
    cout << substituidos << " " << consertados << endl;
    
    return 0;
}
